Gemfile

```diff
@@ -81,7 +81,7 @@ group :test do
 end
 
 platforms :ruby do
-  gem 'nokogiri', '>= 1.6.7.rc3'
+  gem 'nokogiri', '>= 1.6.7'
 
   # Needed for compiling the ActionDispatch::Journey parser.
   gem 'racc', '>=1.4.6', require: false
```

Comment:
Yeah, it's unrelated, but I had to do it to get nokogiri to compile on my machine (see separate commit message).

I can back it out if you prefer.